### DISCUSSION TITLE
117 HD Support cannot find GPU

### DISCUSSION THREAD

**Discussion by Amnesiapc** [runelite/runelite#16069] [Q&A] [answered]
Hey my buddy has been trying to play osrs but the 117 hd plugin wont seem to work and neither does the gpu plugin when he enables it it turns off straight away 

he has an amd radeon 5700 xt ill add the client.logs
[client(2).log](https://github.com/runelite/runelite/files/10303498/client.2.log)

**Comment by Mark7625** [comment, marked as answer]
Hey it seems like you pc doesn't have a gpu 'GDI Generic' if you need more support please join discord.gg/U4p6ChjgSE or make an issue here https://github.com/117HD/RLHD. If you have a gpu it could be the following, It's not Plugged in, You need to update your drivers or you have disabled the Gpu by accident in Windows
